akka-docs/rst/general/addressing.rst

```diff
@@ -143,6 +139,19 @@ systems or JVMs. This means that the logical path (supervision hierarchy) and
 the physical path (actor deployment) of an actor may diverge if one of its
 ancestors is remotely supervised.
 
+
+Actor path alias or symbolic link?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As in some real file-systems you might think of a "path alias" or "symbolic link”for an actor,
+i.e. one actor may be reachable using more than one path.
+However, you should note that actor hierarchy is different from file system hierarchy.
+You cannot freely create actor paths like symbolic links to refer to arbitrary actors.
+As described in the above logical and physical actor path sections,
+an actor path must be either logical path which represents supervision hierarchy, or
+physical path which represents actor deployment. If you could freely create a symbolic link
+actor path, that breaks this rule.
```

Comment:
Since the “rule” is more like “whatever Akka internals happen to require” I would recommend to remove the last sentence.